Sources/Rendering/Core/Glyph3DMapper/Constants.js

```diff
@@ -1,6 +1,7 @@
 export const OrientationModes = {
   DIRECTION: 0,
   ROTATION: 1,
+  MATRIX: 2,
```

Comment:
@martinken we added a new way to input orientation to the `3DGlyphMapper` (3x3 matrix). Could you give this commit a look when you get a chance? It's then used by the `CircleContextRepresentation` in [this commit](https://github.com/Kitware/vtk-js/pull/1422/commits/c50bc37733e63ad71c5f35a4cd5c5dc5de634fa7). Thank you!